### PR TITLE
Function to retrieve PubMedIDs for given PubChem CID

### PR DESCRIPTION
**Pull request by samuelbunga**
This PR consists of a new function that takes PubChem CID as an input and returns back a list of PMIDs corresponding to it.
```
from indra.databases.pubchem_client import get_pmids 
pmids = get_pmids('22442') # aspirin
```

### CODE REVIEW

File: indra/databases/pubchem_client.py [hunk 1]
```diff
@@ -74,3 +74,28 @@ def get_preferred_compound_ids(pubchem_cid):
                 pref_ids |= set(pref_cpd['Value']['Number'])
     pref_ids = sorted([str(pid) for pid in pref_ids])
     return pref_ids
+
+
+def get_pmids(pubchem_cid):
```
Comment (cthoyt) [inline indra/databases/pubchem_client.py:79]:
```suggestion
def get_pmids(pubchem_cid: str) -> List[str]:
```

good time to check that you're explicitly handling integer pmids

Comment (samuelbunga) [reply indra/databases/pubchem_client.py:79]: Thanks Charlie! will make the changes now

File: indra/databases/pubchem_client.py [hunk 2]
```diff
@@ -74,3 +74,28 @@ def get_preferred_compound_ids(pubchem_cid):
...
+    list
+        PubMedIDs corresponding to the given PubChem CID. If none present,
+        an empty list is returned.
+    """
+    url = '%s/compound/cid/%s/xrefs/PubMedID/JSON' % \
+          (pubchem_url, pubchem_cid)
+    res = requests.get(url)
+    if res.status_code != 200:
+        logger.error('Could not retrieve PMIDs for %s' % pubchem_cid)
+        return None
```
Comment (cthoyt) [inline indra/databases/pubchem_client.py:98]: So the return type is `Optional[List[str]]`? Or would it be better to return an empty list?

Comment (bgyori) [reply indra/databases/pubchem_client.py:98]: Please return an empty list here

**Comment by bgyori** [comment]
@samuelbunga thanks, this looks good but the PR is from your master branch, could you please start a branch and reopen the PR from there? You can also add 
```
from indra.databases.pubchem_client import get_pmids 
pmids = get_pmids('22442') # aspirin
```
as a new test case in indra/tests/test_pubchem_client.py.

**Pull request closed without merging**